src/providers/index.ts

```diff
@@ -208,7 +212,7 @@ async function getChildTokenDetails(
     )
     const decimals =
       childToken && chainId === ChainId.BNB
```

Comment:
should we use the decimals from the local_mappings for AVAX too? we had to do this for BNB because some tokens had different decimals on BNB vs mainnet, but unsure if thats also the case for AVAX. I guess it doesnt hurt to do the same for AVAX, in which case this should be `(chainId === ChainId.BNB || chainId === ChainId.AVAX)`

Reply:
GREAT catch thank you